language/use-cases/sensitive-data-detection/context_aware_sensitive_data_detection.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        model_name="text-bison@001",
</pre><p>Make model_name a variable to make it easier to edit.</p><p> </p><p>Also, I'd suggest trying with <code>text-bison@002</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>